ui/src/app/cohort-review/participant-detail/participant-detail.component.css

```diff
@@ -0,0 +1,8 @@
+.participant-browser-header {
+    margin: 0px;
+    flex: 1 1 0%;
```

Comment:
As mentioned, I think we can drop this without affecting anything.

Reply:
Removed.